lib/query/package-features.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash')
+
+function packageFeatures (features = [], { groupBy, aggregates, collection } = {}) {
+  if (groupBy || (!aggregates && !collection)) {
+    return features
+  }
+
+  if (aggregates) {
+    return features[0]
+  }
+
+  if (collection) {
+    return _.chain(collection).cloneDeep().set('features', features).value()
```

Comment:
If the `collection` could come with a long array of features, I would suggest dropping the `features` before a deep clone.

Reply:
`collection` won't have `features` here.  `features` are removed from any collection wrapper eariler, here https://github.com/koopjs/winnow/pull/184/files/d33032c27e29027f1e85bacd7935776d7417b799#diff-a3fdc082f7c80809c8615ac5bfcc56cb562978d7e571bb4ba6e47e3341ab71bcR24-R29